.github/CONTRIBUTING.md

```diff
@@ -1,29 +1,35 @@
 # Prerequisites
+
 - [rust] toolchain
 
 [rust]: https://www.rust-lang.org
 
 # Development
+
 Put those settings inside vimrc,
+
 ```vim
 let g:LanguageClient_devel = 1 " Use rust debug build
 let g:LanguageClient_loggingLevel = 'DEBUG' " Use highest logging level
 ```
 
-1. Make necessary changes.
-1. Build. `make` to build, format and run [clippy], or `make build` to run build only.
-1. Verify changes.
-1. Run tests. (See below section)
+- Make necessary changes.
```

Comment:
It's easier to reference the item when they're numbered. But I'm fine unifying list as well.